cupy/cuda/cufft.pyx

```diff
@@ -35,9 +34,13 @@ cdef enum:
 
 
 cdef extern from 'cupy_cufft.h' nogil:
+    # we duplicate some types here to avoid cimporting from driver/runtime,
+    # as we don't include their .pxd files in the sdist
+    ctypedef void* Stream 'cudaStream_t'
+    ctypedef int DataType 'cudaDataType'
```

Comment:
These are not going to be changed anytime soon so it should be ok,
but I don't like this. However, I understand there is no other way of dealing with this.